### ISSUE TITLE
rclone mount issue

### ISSUE THREAD

**Issue by veritas2884** [ncw/rclone#905]
Hello,
I am running Ubuntu Server LTS 16.04.1 with rclone latest stable.
I have setup services through ACD.
rclone copy works great, but I am having trouble with rclone mount.
Command Run:
sudo rclone mount remote: /home/plex/.acd --allow-other &
Output:
[2] 2072 (sometimes this is different)
Issue:
After, this happens the next command I run, no matter what it is, completes but also is accompanied by this output:
[3]+ Stopped sudo rclone mount remote: /home/plex/acd --allow-other
I have only once got it to display the content of my amazon cloud drive. Now it wont mount again. Any help would be greatly appreciated.

**Comment by ncw** [comment]
This sounds like it is something to do with sudo and job control. 

Two things you could try... 

1) sudo -i then run the rclone command without sudo

2) try the rclone mount without sudo - that should work

**Comment by veritas2884** [comment]
I will try the first suggestion once I am home again. But if I run without sudo it gives me a permission error.

**Comment by veritas2884** [comment]
Thank you for the suggestions

**Comment by veritas2884** [comment]
After I run sudo -i and then the command I get:

2016/11/23 02:46:19 Failed to load config file "/root/.rclone.conf" - using defaults: open /root/.rclone.conf: no such file or directory
2016/11/23 02:46:19 Failed to create file system for "remote:": didn't find section in config file

If I try to run it without sudo I get:

2016/11/23 02:48:09 Failed to load config file "/home/plex/.rclone.conf" - using defaults: open /home/plex/.rclone.conf: permission denied
2016/11/23 02:48:09 Failed to create file system for "remote:": didn't find section in config file

**Comment by ladude626** [comment]
Did you run "rclone config" to create a new remote?

**Comment by ncw** [comment]
Which user did you use to configure rclone?  Either run the mount using that user, or reconfigure using the plex user.

**Comment by veritas2884** [comment]
@ncw I followed these steps to configure rclone while logged in as the plex user:
Fetch and unpack

curl -O http://downloads.rclone.org/rclone-current-linux-amd64.zip
unzip rclone-current-linux-amd64.zip
cd rclone-*-linux-amd64
Copy binary file

sudo cp rclone /usr/sbin/
sudo chown root:root /usr/sbin/rclone
sudo chmod 755 /usr/sbin/rclone
Install manpage

sudo mkdir -p /usr/local/share/man/man1
sudo cp rclone.1 /usr/local/share/man/man1/
sudo mandb 
Run rclone config to setup. See rclone config docs for more details.

rclone config

**Comment by veritas2884** [comment]
@ladude626 @ncw I deleted the remote I already had and re-added a new remote ACD. 

I ran the rclone config (no sudo) to set it up. Then I ran:

rclone mount remote: /home/plex/.acd &

And after about 5 seconds it printed:
2016/11/24 13:54:53 Fatal error: failed to mount FUSE fs: fusermount: exec: "fusermount": executable file not found in $PATH

**Comment by veritas2884** [comment]
So I did a fresh install today of the OS and reset everything. After I installed FUSE and rclone I was able to see all of my ACD directories and eveything worked great.

Upon rebooting the server I again get this:
Command:
sudo rclone mount remote: /home/plex/.acd --allow-other &
Output:
[1] 1525

Next Command
ls /home/plex/.acd

Output:
[1]+  Stopped                 sudo rclone mount remote: /home/plex/.acd --allow-other

So it will not mount again now.
